steward/tests/acceptance.rs

```diff
@@ -30,65 +35,436 @@ use steward::config::StewardConfig;
 /// invocations as `cargo test` executes tests in parallel by default.
 pub static RUNNER: Lazy<CmdRunner> = Lazy::new(|| CmdRunner::default());
 
-/// Use `StewardConfig::default()` value if no config or args
+/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_no_args() {
+fn eth_keys_add() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
+    let config_dir = TempDir::new("test_key")?;
+    let config_file_path = config_dir.path().join("config.toml");
+    let mut f = File::create(config_file_path.clone())?;
+    let configu = StewardConfig {
```

Comment:
I don't think you need to specify each section individually. Instead, you could `let config = StewardConfig::default()` and then replace the individual `keys` and `keystore` sections.

Reply:
I don't think it's possible to use `StewardConfig::default()` and also replace the `keys` and `keystore` section only. However, I think I can achieve something similar to this with `..Default::default()`

Reply:
This should resolve it 48d5f6c3b6f302f43e0405daae23b19890c5db90